src/lib/util/regexes.ts

```diff
@@ -30,6 +30,6 @@ export const mentionableRegexp = /^(?:<@!?)?(?:<@&?)?([0-9]+)>?$/;
 
 /**
  * Regex for command and option names
- * @raw `/^[\P{Lu}\p{N}_-]+$/u`
+ * @raw `/^[a-zA-Z1-9]/`
  */
-export const commandAndOptionNameRegexp = /^[\P{Lu}\p{N}_-]+$/u;
+export const commandAndOptionNameRegexp = /[a-zA-Z1-9-\s]/;
```

Comment:
I'm not sure if the `-` is needed before the `\s`, I am not a regex god so I wouldn't know. If it works its fine